src/formatDate.js

```diff
@@ -50,7 +50,28 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , oldSeparator] = fromFormat;
+  const [, , , newSeparator] = toFormat;
+
```

Comment:
for this case instead of using destruction - better get value by index
```suggestion
  const oldSeparator = fromFormat[3];
  const newSeparator = toFormat[3];

```